src/Centreon/Domain/Repository/HostServiceRelationRepository.php

```diff
@@ -33,6 +33,20 @@ public function export(array $pollerIds): array
 WHERE hr.nagios_server_id IN ({$ids})
 GROUP BY t.hsr_id
 SQL;
+
+        // Extract BA services relations
+        if ($ba) {
+            foreach ($ba as $key => $val) {
+                $ba[$key] = "'ba_{$val}'";
+            }
+              $ba = implode(',', $ba);
+              $sql .= " UNION SELECT t2.*"
+                  . " FROM host_service_relation AS t2"
+                  . " INNER JOIN service s2 ON s2.service_id = t2.service_service_id"
+                  . " AND s2.service_description IN({$ba}) GROUP BY t2.service_service_id";
+        }
+        $sql .= ") AS l GROUP BY l.hsr_id";
```

Comment:
you need to set 'SELECT l.* FROM (' before the row 27

Reply:
done

Reply:
done